src/Data/Lens/At.purs

```diff
@@ -15,6 +16,25 @@ import Data.Newtype (unwrap)
 import Data.Lens (Lens', lens)
 import Data.Lens.Index (class Index)
 
+
+-- | `At` is useful for types like `Data.Map`:
+-- | 
+-- | * The result of getting an element is a `Maybe a`. 
```

Comment:
Do you think we need to say this specifically, when it's also apparent from the type? Documenting the why of things and providing examples seems a more useful form of explanation that reproducing the type as words to me generally, but I'd be interested to hear if you have thoughts counter to that.

Reply:
I changed the text.